omas/omas_physics.py

```diff
@@ -478,7 +478,7 @@ def summary_heating_power(ods, update=True):
     for key, value in power_dict.items():
         power_dict[key].append(numpy.trapz(q_dict[key], x=vol))
         if numpy.sum(value) > 0:
-            if key is 'total_heating':
+            if 'total_heating' in key:
                 ods_n['summary.global_quantities.power_steady.value'] = numpy.array(value)
                 continue
             elif key is 'fusion':
```

Comment:
make sure you get all of these: `key is 'fusion'`

Reply:
I think I caught it in one of my follow up commits